src/main/java/edu/gatech/cs2340/trydent/OneTimeEvent.java

```diff
@@ -0,0 +1,75 @@
+package edu.gatech.cs2340.trydent;
+
+
+/**
+ * Creates an event that starts, updates, and stops exactly once.
+ *
+ */
+public abstract class OneTimeEvent {
```

Comment:
This entire class is duplicate functionality of the TrydentEngine.runLater function (perhaps that should be renamed to TrydentEngine.runOnce to make that more obvious).